pub/index.html

```diff
@@ -0,0 +1,168 @@
+<!doctype html>
+<html>
+  <head>
+    <title>south park blog</title>
+    <style>
+    body {
+      margin: 0px;
+      padding: 0px;
+    }
+    header {
+      background-image: url(http://www.importantstudio.com/ironyard/spbusstop.jpg);
+      margin: 0;
+      width: 1280px;
+      height: 350px;
+      margin: 0px;
+    }
+    .container {
+        width: 850px;
+        padding: 10px;
+        margin: 0 auto 0 auto;
+        height: 1000px;
+
+    }
+    aside {
+      width: 250px;
+      padding: 0 10px 0 10px;
+      float: left;
+
+    }
+    .logodiv {
+      width: 100px;
+      height: 100px;
+      position: relative;
+      top: -70px;
+      border-radius: 50%;
+      background: url(http://www.importantstudio.com/ironyard/sps.jpg) center center no-repeat;
+      border: 4px solid white;
+    }
+    article {
+      width: 520px;
+      padding: 5px 0 0 0;
+      float: right;
+      margin: 5px 0px 5px 0px;
+
+    }
+    .fixed {
+      width: 200px;
+      height: 20px;
+      position: fixed;
+      background: #37FDFC;
+      top: 5px;
+      right: 5px;
+      margin: 0;
+      padding: 3px;
+      border-radius: 5px;
+      box-shadow: 3px 3px 3px #000000;
+
+    }
```

Comment:
While this class name does convey the _style_ that's applied to the element, it might be better to name it according to the _purpose_ of the element instead.